### ISSUE TITLE
Change url Nude-moon

### ISSUE THREAD

**Issue by Gostik21** [keiyoushi/tachiyomi-extensions-source#5291]
### Source information

Nude-moon 1.4.18

### Source language

Russian

### Source new URL

https://b.nude-moon.fun

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL has not changed yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by vetleledaal** [comment]
Do the images on the site load for you? I can't get it to work. They might be blocking connections outside of Russia.

**Comment by Gostik21** [comment]
Yes, the images are loading for me, but I have to use a VPN because tachiemi can't get past cloudflare. But after the previous URL update it worked for a while

**Comment by Gostik21** [comment]
It would be great to add the ability to change the URL to the extension itself, like in other extensions.
